cdc/owner_test.go

```diff
@@ -347,10 +353,10 @@ func (s *ownerSuite) TestDDL(c *check.C) {
 func (s *ownerSuite) TestHandleAdmin(c *check.C) {
 	cfID := "test_handle_admin"
 	sampleCF := &changeFeed{
-		ID:               cfID,
-		info:             &model.ChangeFeedInfo{},
-		ChangeFeedStatus: &model.ChangeFeedStatus{},
-		State:            model.ChangeFeedSyncDML,
+		ID:       cfID,
+		info:     &model.ChangeFeedInfo{},
+		status:   &model.ChangeFeedStatus{},
+		DDLState: model.ChangeFeedSyncDML,
```

Comment:
It seems `DDLState` is not a suitable name.

Reply:
what suggestion do you have?